backend/games/word_guesser/ClassicWordle.py

```diff
@@ -0,0 +1,137 @@
+from random import randint
+from typing import Dict, List
+
+
+class ClassicWordle:
+    # maximum number of guesses
+    MAX_GUESSES = 6
+    WORD_LIST = [
+        "adieu",
+        "odium",
+        "shade",
+        "resin",
+        "alert",
+        "haunt",
+        "orate",
+        "media",
+        "blind",
+        "route",
+        "audio",
+        "pause",
+        "alien",
+        "canoe",
+        "plane",
+        "rouse",
+        "fraud",
+        "atone",
+        "raise",
+        "minor",
+    ]
+
+    def __init__(self):
+        # represents the state of the game, index 0 will hold the "goal word"
+        game_board = List[str]
```

Comment:
This is not a valid initialization, the type hint and initial value need to be defined separately, because the type is just a hint, not a value:

```python
game_board: List[str] = []
```

This shows how we add a colon after the variable name to add the type hint, then put the value after equals sign.